diff --git a/templates/block/block--system-branding-block.html.twig b/templates/block/block--system-branding-block.html.twig
index 9538626..f45b060 100644
--- a/templates/block/block--system-branding-block.html.twig
+++ b/templates/block/block--system-branding-block.html.twig
@@ -17,11 +17,11 @@
 {% block content %}
   {% set site_logo_url = site_logo ? file_url(site_logo) : '' %}
 
-  <a href="{{ home_url|default('/') }}" class="inline-block" rel="home">
+  <a href="{{ home_url|default('/') }}" class="inline-block" rel="home" aria-label="{{ site_name ? site_name : 'Home'|t }}">
     {% block branding %}
       {% if site_logo %}
         <div class="branding flex items-center">
-          <img class="h-full w-full" src="{{ site_logo_url }}" alt="{{ site_name }}" />
+          <img class="h-full w-full" src="{{ site_logo_url }}" alt="" />
         </div>
       {% endif %}
       {% if site_name %}
